src/Discord/Provider.php

```diff
@@ -61,6 +61,24 @@ protected function getUserByToken($token)
         return json_decode($response->getBody()->getContents(), true);
     }
 
+    /**
+     * @param array $user
+     *
+     * @return string|null
+     */
+    protected function formatAvatar(array $user)
+    {
+        if (is_null($user['avatar'])) {
```

Comment:
```suggestion
        if (empty($user['avatar'])) {
```

Reply:
I will get this changed, just carried on with what was already in the file

Reply:
thanks :)